Task-6/Task-6/Services/PhonePurchase.cs

```diff
@@ -0,0 +1,115 @@
+using NLog;
+using Task_6.Exceptions;
+using Task_6.Models;
+
+namespace Task_6.Services;
+
+public static class PhonePurchase
+{
+    private static Logger _logger = LogManager.GetCurrentClassLogger();
```

Comment:
For every place, where you are creating instance of Logger. 
Just create Log class with static method Info(string msg).
And then you call it everywhere Log.Info("Blablabla")